.github/workflows/ci.yml

```diff
@@ -587,6 +587,10 @@ jobs:
         uses: actions/download-artifact@v4
         with:
           merge-multiple: true
+      # Having multiple artifacts named the same might be onfusing to the users. Drop the unsigned versions
+      - name: Remove unsigned artifacts
+        run: |
+          rm -rf *unsigned*
```

Comment:
Should we add an if to do this only for windows?

Reply:
No need in my opinion. All the artifacts are collected in this job and only matches would be the ones to be removed